### ISSUE TITLE
MSON - serialize "samples" and "default" attributes in refract form instead simple value

### ISSUE THREAD

**Issue by klokane** [apiaryio/drafter#100]
```mson
- a: 3,4 (array, sample)
```

is now serialized with "samples" in non-refracted form
```json
"samples" : [["3","4"]]
```

It should be serialized in same form as content:
```json
"samples": [
  [{
    "element": "string",
     "content": "3"
  },
  {
    "element": "string",
     "content": "4"
  }],
]
```

**Comment by pksunkara** [comment]
Yes. This should be fixed before release.

**Comment by pksunkara** [comment]
Fixed by #104
